test/controllers/v1/profiles_controller_test.rb

```diff
@@ -11,28 +11,65 @@ class ProfilesControllerTest < ActionDispatch::IntegrationTest
       profiles(:one).update! account: accounts(:test)
     end
 
-    test 'tailoring_file with a canonical profile returns no content' do
-      profiles(:one).update! rules: [rules(:one)]
-      get tailoring_file_v1_profile_url(profiles(:one).id)
-      assert_response :no_content
-    end
+    class TailoringFileTest < ProfilesControllerTest
+      test 'tailoring_file with a canonical profile returns no content' do
+        profiles(:one).update! rules: [rules(:one)]
+        get tailoring_file_v1_profile_url(profiles(:one).id)
+        assert_response :no_content
+      end
+
+      test 'tailoring_file with a noncanonical profile matching its parent'\
+        'returns no content' do
+        profiles(:two).update!(rules: [rules(:one)])
+        profiles(:one).update!(parent_profile: profiles(:two),
+                               rules: [rules(:one)])
+        get tailoring_file_v1_profile_url(profiles(:one).id)
+        assert_response :no_content
+      end
 
-    test 'tailoring_file with a noncanonical profile matching its parent'\
-      'returns no content' do
-      profiles(:two).update!(rules: [rules(:one)])
-      profiles(:one).update!(parent_profile: profiles(:two),
-                             rules: [rules(:one)])
-      get tailoring_file_v1_profile_url(profiles(:one).id)
-      assert_response :no_content
+      test 'tailoring_file with a noncanonical profile '\
+           'returns tailoring file' do
+        profiles(:two).update!(rules: [rules(:one), rules(:two)])
+        profiles(:one).update!(parent_profile: profiles(:two),
+                               rules: [rules(:one)])
+        get tailoring_file_v1_profile_url(profiles(:one).id)
+        assert_response :success
+        assert_equal Mime[:xml].to_s, @response.content_type
+      end
     end
 
-    test 'tailoring_file with a noncanonical profile returns tailoring file' do
-      profiles(:two).update!(rules: [rules(:one), rules(:two)])
-      profiles(:one).update!(parent_profile: profiles(:two),
-                             rules: [rules(:one)])
-      get tailoring_file_v1_profile_url(profiles(:one).id)
-      assert_response :success
-      assert_equal Mime[:xml].to_s, @response.content_type
+    class DestroyTest < ProfilesControllerTest
```

Comment:
No need for a subclass here, thoughts about using a `context 'destroy'` block?

Reply:
context is for rspec - this is minitest. The subclasses are nice IMO and allow us to one day even move them out to their own files when this test gets really big, like `test/controllers/v1/profiles_controller/destroy_test.rb`.

Reply:
FWIW context is available in minitest because we use https://github.com/thoughtbot/shoulda-context

Reply:
Right, but this doesn't work with the minitest `test 'foo'` syntax that we're using in all of our controllers. I'm open to changing all of them to `should 'foo'`, though I don't really see the downside of defining classes like this either. Either way, whatever we ultimately decide in the future, we can easily swap between the different syntax.

It seems also that the shoulda-context benefits are mostly the helper methods provided with shoulda-matchers like we're using in our model tests (i.e. `should validate_presence_of(:first_name)`). There are also helpers like this for controllers, but they seem less useful to me.